intranet/templates/polls/poll.html

```diff
@@ -40,10 +40,15 @@ <h3>
         for
         {% for group in poll.groups.all %}
             {{ group }}
-            {% if not forloop.last %},{% endif %}
+            {% if not forloop.last %},
+            {% else %} &bull;
+            {% endif %}
         {% empty %}
-            everyone
+            everyone &bull;
         {% endfor %}
+        {% if is_polls_admin and not poll.before_start_time %}
+            {{ poll.get_percentage_voted }} of eligible users have voted
```

Comment:
This message should be roughly similar to the message in `intranet/templates/polls/results.html` (i.e. incorporate raw numbers). I would suggest making the message a property of a `Poll` that returns the message as a string.